src/module-elasticsuite-catalog-optimizer/Model/Optimizer/RelevantOptimizerList.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * DISCLAIMER
+ * Do not edit or add to this file if you wish to upgrade Smile Elastic Suite to newer
+ * versions in the future.
+ *
+ * @category  Smile
+ * @package   Smile\ElasticsuiteCatalogOptimizer
+ * @author    Romain Ruaud <romain.ruaud@smile.fr>
+ * @copyright 2018 Smile
+ * @license   Open Software License ("OSL") v. 3.0
+ */
+namespace Smile\ElasticsuiteCatalogOptimizer\Model\Optimizer;
+
+/**
+ * Return relevant optimizers for a given search context.
+ *
+ * @category Smile
+ * @package  Smile\ElasticsuiteCatalogOptimizer
+ * @author   Romain Ruaud <romain.ruaud@smile.fr>
+ */
+class RelevantOptimizerList
```

Comment:
The name of the class is not appropriate.
It suggest a that optimizer are selected using a relevance algorithm.

OptimizerList seems more accurate to me.

Reply:
Ok for this one, just OptimizerList is fine for me too.